pkg/config/toml/docs.toml

```diff
@@ -118,6 +118,10 @@ LogBroadcasterEnabled = true # Default
 #
 # Set to zero to disable.
 NoNewFinalizedHeadsThreshold = '0' # Default
+# TxMinimumWaitTimeForConfirmation minimum time to wait for a TX to get into a block in the blockchain. This is used for the EVMService.SubmitTransaction operation. It won't check for TX status updates until this time passes
```

Comment:
Why do we even need a minimum?

Reply:
We need a minimum because if a block takes 12 seconds to be created it doesn't make sense to be checking if it's confirmed before that time.

Reply:
Are there any existing fields that we could derive this from dynamically instead of having to calibrate it for each chain?

Reply:
Although, FWIW the time between submission and inclusion can be arbitrarily small no matter how long the block period is - it just might happen to show up right before the next block goes out.

Reply:
Open to help here!. What do you guys suggest?. Maybe keep TxMinimumWaitTimeForConfirmation very low?. Something like 5 seconds? Less?. And keep TxMaximumWaitTimeForConfirmation to 300 seconds?

Reply:
I think we don't need the minimum time since as Jordan said the inclusion time could be arbitrarily small. I haven't seen this inclusion minimum poll time pattern used anywhere else

Reply:
Then I'll drop the TxMinimumWaitTimeForConfirmation and just keep the TxMaximumWaitTimeForConfirmation which I'll rename to ConfirmationMaxWaitTime per @jmank88  suggestion.  Then I can set a default value of 60 seconds which will work well for most chains and use 5 minutes in EVM which is the maximum it seems it may take. Makes sense?

Reply:
I think that makes sense, although would naming the field `ConfirmationTimeout` make sense in that case?